docs/built-in-bpa-rules.md

```diff
@@ -198,4 +198,9 @@ Transparent data encryption should be enabled to protect data-at-rest and meet c
 Set the data retention for your SQL Server's auditing to storage account destination to at least 90 days.
 
 **Recommendation**: For incident investigation purposes, we recommend setting the data retention for your SQL Server's auditing to storage account destination to at least 90 days, in the [Microsoft.Sql/servers/auditingSettings resource properties](https://docs.microsoft.com/en-us/azure/templates/microsoft.sql/2020-11-01-preview/servers/auditingsettings?tabs=json#serverblobauditingpolicyproperties-object), using the *retentionDays* property. Confirm that you are meeting the necessary retention rules for the regions in which you are operating. This is sometimes required for compliance with regulatory standards.
-#### Severity: 3
\ No newline at end of file
+#### Severity: 3
+
+## TA-000029: Azure API Management APIs should use encrypted protocols only
```

Comment:
Do you want to add a Severity rating for this rule?

Reply:
Yes. I don't see that you put the value in this file. Can you add it?

Reply:
Hey Anais, apologies, was looking at the wrong file. I've added the Severity to this file (Line 207)